src/main/java/alex/jenkins/plugins/FileSystemListParameterDefinition.java

```diff
@@ -175,7 +171,7 @@ public ParameterValue getDefaultParameterValue() {
 		try {
 			defaultValue = getEffectiveDefaultValue();
 		} catch (IOException e) {
-			LOGGER.warning(formatter.format(Messages.FileSystemListParameterDefinition_SymlinkDetectionError(), defaultValue).toString());
+			LOGGER.warning(String.format(Messages.FileSystemListParameterDefinition_SymlinkDetectionError(), defaultValue).toString());

```

Comment:
Silly:

* `Messages` can handle text formatting itself.
* Text printed to a `Logger` should not be localized to begin with.

Sigh.

Reply:
I decided not to touch that in my fix, because in some cases messages go to both System Log in UI.